test_rules_scala.sh

```diff
@@ -883,3 +887,4 @@ $runner bazel build //test_expect_failure/proto_source_root/... --strict_proto_d
 $runner scala_binary_jar_is_exposed_in_build_event_protocol
 $runner scala_test_jar_is_exposed_in_build_event_protocol
 $runner scala_junit_test_jar_is_exposed_in_build_event_protocol
+$runner test_mix_java_scala_lib
```

Comment:
I don’t think we need this. Your test is already a bazel test. This script is for the tests we can’t easily make normal bazel tests.

Reply:
Ah nice, I didn't see the call to `bazel test //...` at first. I'll get rid of this.